src/drivers/boards/aerocore2/board_config.h

```diff
@@ -0,0 +1,256 @@
+#define GPIO_SPI_CS_ACCEL_MAG	(GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTE|GPIO_PIN2)
+#define GPIO_SPI_CS_BARO	(GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTE|GPIO_PIN4)
+#define GPIO_SPI_CS_FRAM	(GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTE|GPIO_PIN11)
+
+#define PX4_SPI_BUS_SENSORS	3
+#define PX4_SPI_BUS_RAMTRON	4
+#define PX4_SPI_BUS_EXT		1
+#define PX4_SPI_BUS_BARO	PX4_SPI_BUS_SENSORS
+
+/* Use these in place of the spi_dev_e enumeration to select a specific SPI device on SPI3 */
+#define PX4_SPIDEV_GYRO		1
+#define PX4_SPIDEV_ACCEL_MAG	2
+#define PX4_SPIDEV_BARO		3
+
+/* I2C busses */
+#define PX4_I2C_BUS_EXPANSION	1
+#define PX4_I2C_BUS_ONBOARD	2
+
+/*
+ * ADC channels
+ *
+ * These are the channel numbers of the ADCs of the microcontroller that can be used by the Px4 Firmware in the adc driver
+ */
+#define ADC_CHANNELS (1 << 10) | (1 << 11) | (1 << 12) | (1 << 13)
+
+/*
+ * ADC defines to be used in sensors.cpp to read from a particular channel
+*/
+#define ADC_BATTERY_VOLTAGE_CHANNEL	10
+#define ADC_BATTERY_CURRENT_CHANNEL	0
+
+/* Define Battery 1 Voltage Divider
+ */
+
+#define BOARD_BATTERY1_V_DIV   (11.0f)
+
+/* Tone alarm output */
+#define TONE_ALARM_TIMER	10	/* timer 10 */
+#define TONE_ALARM_CHANNEL	1	/* channel 1 */
+#define GPIO_TONE_ALARM_IDLE	(GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_CLEAR|GPIO_PORTB|GPIO_PIN8)
+#define GPIO_TONE_ALARM		(GPIO_ALT|GPIO_AF3|GPIO_SPEED_2MHz|GPIO_PUSHPULL|GPIO_PORTB|GPIO_PIN8)
+
+/* PWM
+ *
+ * Eight PWM outputs are configured.
+ *
+ * Pins:
+ *
+ * CH1 : PD12 : TIM4_CH1
+ * CH2 : PD13 : TIM4_CH2
+ * CH3 : PD14 : TIM4_CH3
+ * CH4 : PD15 : TIM4_CH4
+ * CH5 : PA0  : TIM5_CH1
+ * CH6 : PA1  : TIM5_CH2
+ * CH7 : PA2  : TIM5_CH3
+ * CH8 : PA3  : TIM5_CH4
+ */
+#define GPIO_TIM4_CH1OUT	GPIO_TIM4_CH1OUT_2
+#define GPIO_TIM4_CH2OUT	GPIO_TIM4_CH2OUT_2
+#define GPIO_TIM4_CH3OUT	GPIO_TIM4_CH3OUT_2
+#define GPIO_TIM4_CH4OUT	GPIO_TIM4_CH4OUT_2
+#define GPIO_TIM5_CH1OUT	GPIO_TIM5_CH1OUT_1
+#define GPIO_TIM5_CH2OUT	GPIO_TIM5_CH2OUT_1
+#define GPIO_TIM5_CH3OUT	GPIO_TIM5_CH3OUT_1
+#define GPIO_TIM5_CH4OUT	GPIO_TIM5_CH4OUT_1
+#define DIRECT_PWM_OUTPUT_CHANNELS	8
+
+/* PWM
+ *
+ * Eight PWM inputs are configured.
+ *
+ * Pins:
+ *
+ * CH8 : PE5  : TIM9_CH1
+ * CH7 : PE6  : TIM9_CH2
+ * CH6 : PC6  : TIM3_CH1
+ * CH5 : PC7  : TIM3_CH2
+ * CH4 : PC8  : TIM3_CH3
+ * CH3 : PA8  : TIM1_CH1
+ * CH2 : PA9  : TIM1_CH2
+ * CH1 : PA10 : TIM1_CH3
+ */
+#define GPIO_TIM9_CH1IN		GPIO_TIM9_CH1IN_2
+#define GPIO_TIM9_CH2IN		GPIO_TIM9_CH2IN_2
+#define GPIO_TIM3_CH1IN		GPIO_TIM3_CH1IN_3
+#define GPIO_TIM3_CH2IN		GPIO_TIM3_CH2IN_3
+#define GPIO_TIM3_CH3IN		GPIO_TIM3_CH3IN_2
+#define GPIO_TIM1_CH1IN		GPIO_TIM1_CH1IN_1
+#define GPIO_TIM1_CH2IN		GPIO_TIM1_CH2IN_1
+#define GPIO_TIM1_CH3IN		GPIO_TIM1_CH3IN_1
+#define DIRECT_INPUT_TIMER_CHANNELS  8
+
+/* spektrum satellite receiver input */
```

Comment:
POWER_SPEKTRUM(_s) is not defined any longer.

Please See https://github.com/PX4/Firmware/blob/master/src/drivers/boards/px4fmu-v4/board_config.h#L265-L282 for an example of the new naming and usage.